src/Microsoft.CodeQuality.Analyzers/UnitTests/QualityGuidelines/ValidateArgumentsOfPublicMethodsTests.cs

```diff
@@ -3243,5 +3243,35 @@ public void M1(C c, C c2)
 }
 ");
         }
+
+        [Fact, WorkItem(1852, "https://github.com/dotnet/roslyn-analyzers/issues/1852")]
+        public void Issue1852()
```

Comment:
Thanks for the minimal repro @dotpaul - made the investigation so much more easier!!